docs/UserGuide.md

```diff
@@ -73,27 +79,27 @@ Example:
 
 ### Marking as complete : `mark`
 
-Marks specified task from the existing task list from as complete.
-
-Format: `mark INDEX`
-- Marks the task at the specified `INDEX` as complete.
-- The index refers to the index number shown in the displayed task list.
-- The index **must be a positive integer** 1, 2, 3,
+Marks the task at the given index of the existing task list as complete. At least 1 index has to be provided, but
+multiple indexes can be provided as well to mark multiple tasks as complete at a time.  
```

Comment:
I think it may be better to leave sentences in documentation in one line, instead of breaking them up into multiple lines!